Tests/PointFreeTests/DatabaseTests.swift

```diff
@@ -1,26 +1,30 @@
 import Database
+import DatabaseTestSupport
 import GitHub
+import PointFreeTestSupport
 import Prelude
 import Models
 import ModelsTestSupport
 import GitHubTestSupport
 import Logging
 import SnapshotTesting
 import XCTest
+@testable import PointFree
 
-final class DatabaseTests: DatabaseTestCase {
+
+final class DatabaseTests: TestCase {
```

Comment:
Is there no more DatabaseTestCase?

Reply:
Not needed since moving back to PointFreeTests...